src/cf-typography/src/molecules/list.less

```diff
@@ -0,0 +1,103 @@
+//
+// Unstyled list modifier
+//
+
+.m-list__unstyled,
+.m-list__horizontal,
+.m-list__icons,
+.m-list__links {
+    padding-left: 0;
+
+    list-style-type: none;
+
+    // This is needed for dd elements.
+    .a-list-item {
```

Comment:
Ah, that's what what. Because it leaves a lot more room to grow later. Yes it'll always be the child of a list, but by making it an atom we could globally target all item atoms that aren't just the default.